numpy/core/overrides.py

```diff
@@ -109,6 +109,18 @@ def decorator(func):
     return decorator
 
 
+
```

Comment:
Opst, empty line too much (don't worry about it though), just noticed that I did not nitpick ;).

Reply:
Opps